tlsobs-api/main.go

```diff
@@ -71,17 +72,29 @@ func main() {
 		}
 	}()
 
+	middlewares := []Middleware{
+		addRequestID(),
+		addDB(db),
+		logRequest(),
+		setResponseHeaders(),
+	}
+
+	sess, err := session.NewSession()
+	if err != nil {
+		log.Printf("Could not create AWS session: %s", err)
+	} else {
+		cloudwatchSvc := cloudwatch.New(sess)
+		middlewares = append(middlewares, addCloudwatch(cloudwatchSvc))
+		log.Printf("CloudWatch configured.")
```

Comment:
If setting up cloudwatch fails for any reason, we show a warning and keep going. I don't think we need a setting to turn it off if we do this.